charts/README.md

```diff
@@ -0,0 +1,132 @@
+
+OpenEBS Data Engines and Control Plane are implemented as micro-services, deployed as containers and orchestrated by Kubernetes itself. An added advantage of being a completely Kubernetes native solution is that administrators and developers can interact and manage OpenEBS using all the wonderful tooling that is available for Kubernetes like kubectl, Helm, Prometheus, Grafana, etc.
+
+OpenEBS turns any storage available on the Kubernetes worker nodes into local or distributed Kubernetes Persistent Volumes.
+* Local Volumes are accessible only from a single node in the cluster. Pods using Local Volume have to be scheduled on the node where volume is provisioned. Local Volumes are typically preferred for distributed workloads like Cassandra, MongoDB, Elastic, etc that are distributed in nature and have high availability built into them. Depending on the type of storage attached to your Kubernetes worker nodes, you can select from different flavors of Dynamic Local PV - Hostpath, LVM, or ZFS.
+* Replicated Volumes as the name suggests, are those that have their data synchronously replicated to multiple nodes. Volumes can sustain node failures. The replication also can be setup across availability zones helping applications move across availability zones. Depending on the type of storage attached to your Kubernetes worker nodes and application performance requirements, you can select Mayastor.
+
+## Documentation and user guides
+
+You can run OpenEBS on any Kubernetes 1.21+ cluster in a matter of minutes. See the [Quickstart Guide to OpenEBS](https://openebs.io/) for detailed instructions.
+
+## Getting started
+
+### Dependencies
+
+OpenEBS helm chart is an umbrella chart that pulls together engine specific charts. The engine charts are included as dependencies.
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://openebs.github.io/dynamic-localpv-provisioner | localpv-provisioner | 3.5.0 |
+| https://openebs.github.io/lvm-localpv | lvm-localpv | 1.4.0 |
+| https://openebs.github.io/mayastor-extensions | mayastor | 2.5.0 |
+| https://openebs.github.io/zfs-localpv | zfs-localpv | 2.4.0 |
+
+```bash
+openebs
+├── (default) LocalPV HostPath
+├── (default) LocalPV LVM
+├── (default) LocalPV ZFS
+└── (default) Mayastor (replicated)
+```
+
+### Prerequisites
+
+- Kubernetes 1.18+ with RBAC enabled
+- Depending on the engine and type of platform, you may have to customize the values or run additional pre-requisites. Refer to [documentation](https://openebs.io).
+
+### Setup Helm Repository
+
+Before installing OpenEBS Helm charts, you need to add the [OpenEBS Helm repository](https://openebs.github.io/charts) to your Helm client.
+
+```bash
+helm repo add openebs https://openebs.github.io/charts
+helm repo update
+```
+
+### Installing OpenEBS
```

Comment:
Should we include upgrade from previous chart here as well?

Reply:
I don't think we have any such content at the moment.

Reply:
We will provide the link to the doc once its ready.